sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwapWithNamedConventionsExamples.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using LusidFeatures;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Instruments
+{
+    [TestFixture]
+    public class InterestRateSwapWithNamedConventions: DemoInstrumentBase
```

Comment:
Whether you want to do it now or later, this class probably needs additional documentation. I don't think the meaning of "with named conventions" is necessarily obvious to everyone who reads it

Reply:
I copied the naming from existing file tbh. 
Most definitely be revisiting this add tests and touch up!